packages/firebase_auth/firebase_auth_web/lib/src/interop/auth.dart

```diff
@@ -33,7 +33,7 @@ Auth getAuthInstance(App app) {
       app.jsObject,
       auth_interop.AuthOptions(
         errorMap: auth_interop.debugErrorMap,
-        persistence: persistences.toJS,
+        persistence: customJSList(persistences),
```

Comment:
Do we need to do that, the customJSList will probably need to be removed when we remove the package:js dependency does `persistences.toJS as JSAny` not enough?

Reply:
<img width="509" alt="Screenshot 2024-02-22 at 09 31 25" src="https://github.com/firebase/flutterfire/assets/16018629/9d1afde5-c5c9-4d18-b3b8-c2865f507900">

toJS doesn't appear to work for List types. We were using `jsify` before we used type feature: https://github.com/firebase/flutterfire/pull/12338/files#diff-073c6ae89d1947a587a79d2020e2cd9478a0d218a4aeb8dd6bce4b37344b92e1L26-L36

Reply:
Let's go, I've tried with JSBox but doesn't work as well :)

Reply:
It should work once the minimum SDK version is bumped to `3.3.0`. See: https://github.com/firebase/flutterfire/pull/12338/files#diff-073c6ae89d1947a587a79d2020e2cd9478a0d218a4aeb8dd6bce4b37344b92e1R36

But for now, this will work 😄